docs/deployment/deployK8sClusterMetrics.md

```diff
@@ -87,3 +87,57 @@ The `Ping Identity Overview` dashboard will have a dropdown for namespace at the
 ![](../images/cluster-metrics-dashboard.png)
 
 Any of the panels can be edited, or new ones created to fit your needs.
+
+## HorizontalPodAutoscaler
+
+Provided that you use the `autoscaling/v2` API version, you can configure a HorizontalPodAutoscaler to scale based on a custom metric (that is not built in to Kubernetes or any Kubernetes component).
+If you are using our [Helm Charts](https://github.com/pingidentity/helm-charts), you can pass the custom metrics under `global.cluster.autoscalingMetricsTemplate`
```

Comment:
....MetricsTemplate'.  The example code here will scale on a requests-per-second threshold of 10,000: